google/cloud/forseti/enforcer/gce_firewall_enforcer.py

```diff
@@ -59,6 +59,10 @@
 # The number of times to retry an operation if it times out before completion.
 OPERATION_RETRY_COUNT = 5
 
+# Name restrictions described at
+# https://cloud.google.com/compute/docs/reference/rest/v1/firewalls
+VALID_RESOURCE_NAME_RE = re.compile('(?:^[a-z](?:[-a-z0-9]{0,61}[a-z0-9])?$)')
```

Comment:
Might need to define or import the `re` package here- the Travis build is spitting out errors:

```ImportError: Failed to import test module: enforcer.project_enforcer_test
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/unittest/loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "/usr/local/lib/python3.6/unittest/loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "/home/forseti/forseti-security/tests/enforcer/project_enforcer_test.py", line 32, in <module>
    from google.cloud.forseti.enforcer import project_enforcer
  File "/home/forseti/forseti-security/google/cloud/forseti/enforcer/project_enforcer.py", line 30, in <module>
    from google.cloud.forseti.enforcer import gce_firewall_enforcer as fe
  File "/home/forseti/forseti-security/google/cloud/forseti/enforcer/gce_firewall_enforcer.py", line 64, in <module>
    VALID_RESOURCE_NAME_RE = re.compile('(?:^[a-z](?:[-a-z0-9]{0,61}[a-z0-9])?$)')
NameError: name 're' is not defined```

Reply:
Ack, added re